WordPress/Classes/ViewRelated/Comments/CommentsViewController.m

```diff
@@ -431,10 +430,13 @@ - (void)syncHelper:(WPContentSyncHelper *)syncHelper syncContentWithUserInteract
                                             });
                                         }
 
-                                        if (![self isTableViewEmpty]) {
-                                            NSString *title = NSLocalizedString(@"Unable to Sync", @"Title of error prompt shown when a sync the user initiated fails.");
-                                            [WPError showNetworkingAlertWithError:error title:title];
-                                        }
+                                        dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.2 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
+                                            [weakSelf refreshPullToRefresh];
```

Comment:
I think this is no needed, right?

Reply:
I think it is, `refreshPullToRefresh` just ends the refreshControl animations if it is currently animating.